tests/Unit/Product/BusinessUnitTest.php

```diff
@@ -216,4 +262,109 @@ public function testItReturnsAJsonRepresentation(): void
 
         $this->assertJsonStringEqualsJsonString($expectedJson, Json::encode($businessUnit));
     }
+
+    public function testItThrowsAExceptionWithoutAOperatorCodeInTheXml(): void
+    {
+        $this->expectException(InvalidXmlStructureException::class);
+
+        $this->expectExceptionMessage('The xml structure is not valid for a BusinessUnit. The property OperatorCode should exist');
+
+        $xml = '
+          <BusinessUnit>
+            <BusinessUnit>Falabella</BusinessUnit>
+            <Price>1500.00</Price>
+            <SpecialPrice>1200.00</SpecialPrice>
+            <SpecialFromDate>2020-12-01 00:00:00</SpecialFromDate>
+            <SpecialToDate>2020-12-30 00:00:00</SpecialToDate>
+            <Stock>15</Stock>
+            <Status>active</Status>
+            <IsPublished>0</IsPublished>
+          </BusinessUnit>';
```

Comment:
Same as `BusinessUnitTest.php `  
You can use de `getSchema` method (LinioTestCase.php)